src/APICalls.js

```diff
@@ -0,0 +1,7 @@
+
+function getAPIData(info) {
+  return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/${info}`)
+  .then((res) => res.json())
+}
```

Comment:
Nice! I love that this fetch it reusable! I wonder if we could be a bit more descriptive - both in the function name, and the parameters name. What kind of data? What does it represent? What kind of info? What does it represent?